### PR TITLE
Rename boldrefs to distortedrefs

### PR DESCRIPTION
**Pull request by mattcieslak**
The BOLD-specific inputs and outputs are renamed to be more generic and to specify whether the image is distorted or corrected.

### CODE REVIEW

File: sdcflows/workflows/base.py
Comment (oesteban) [inline sdcflows/workflows/base.py]: why the change to indexed access? Isn't pybids deprecating it?
```suggestion
        if associated.suffix in ('epi', 'phasediff', 'fieldmap', 'phase1', 'phase2'):
```

**Review by oesteban — COMMENTED** [review]
Very much appreciated. Left one small question.

**Comment by mattcieslak** [comment]
I haven't kept up with pybids, you're probably right.

**Review by oesteban — APPROVED** [review]
Sorry I had to rebase your PR, will merge right after tests pass.

**Comment by oesteban** [comment]
Actually, I just realized some changes from #30 leaked in this one. Will fix asap.

**Review by oesteban — APPROVED** [review]
Okay, done. Thanks!

**Pull request merged**
